packages/flutter_tools/lib/src/commands/test.dart

```diff
@@ -216,6 +216,13 @@ class TestCommand extends FlutterCommand with DeviceBasedDevelopmentArtifacts {
               'as a multiplier of the default timeout (e.g. "2x"), '
               'or as the string "none" to disable the timeout entirely.',
         defaultsTo: '30s',
+      )
+      ..addFlag('enable-fractional-translation',
+        help: 'Enables fraction translation of engine composited layers in '
+              'the software renderer backend for flutter tester. Also disables '
+              'the raster cache.',
+        defaultsTo: true,
```

Comment:
Will this break all open source customers' goldens?

Reply:
if it doesn't break customer-tests, then no :)